cmd/root/bundle.go

```diff
@@ -94,6 +95,8 @@ func configureBundle(cmd *cobra.Command, b *bundle.Bundle) (*bundle.Bundle, diag
 		return b, diags
 	}
 
+	diags = diags.Extend(bundle.Apply(ctx, b, validate.NoInterpolationInAuthConfig()))
```

Comment:
You can actually convert the whole `NoInterpolationInAuthConfig` mutator in a usual Go function and don't use bundle.Apply here at all because this mutator does not actually mutate anything